docs/scos/dev/architecture/module-api/definition-of-module-api.md

```diff
@@ -22,48 +22,49 @@ redirect_from:
   - /v2/docs/en/definition-api
   - /v1/docs/definition-api
   - /v1/docs/en/definition-api
+  - /docs/scos/dev/architecture/module-api/definition-of-module-api.html
 ---
 
-According to [Semantic Versioning](http://semver.org/), Spryker releases a major version “when there are incompatible API changes”. To make a sound decision about the version type, we need to define our internal APIs.
+According to [Semantic Versioning](http://semver.org/), we release a major version of a module when there are backward compatibility(BC) breaking changes in the Public API. This document declares what public and private APIs are.
 
-In the Spryker Commerce OS’s core, all public methods in theses locatable classes are considered as API:
+## Public API
 
-* [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
-* [Clients](/docs/scos/dev/back-end-development/client/client.html)
-* [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
-* [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
-* Plugin defining protected methods in dependency providers.
+In the Spryker Commerce OS’s core, the following is the public API:
 
-And the interfaces which are implemented everywhere are also part of the API:
+* Public methods in these locatable classes:
+    * [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
+    * [Clients](/docs/scos/dev/back-end-development/client/client.html)
+    * [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
+    * [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
+    * Plugin defining protected methods in dependency providers.
 
-* Plugin interfaces
-* Plugins
-
-In addition to these obvious cases, there are some other classes that are part of the API and can cause a BC break:
-
-* module Config [`Client/Yves/Zed/Shared/Service`](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
-* Controllers
-* Twig functions
-* [CLI commands](/docs/scos/dev/back-end-development/console-commands/implementing-a-new-console-command.html)
-* Public constants that define environment configuration in [Constant Interfaces](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
-
-Every change in a schema can cause a BC break:
+* Interfaces:
+    * Plugin interfaces
+    * Plugins
 
+* Other classes:
```

Comment:
The database is not a class. 
Shall we rename to something like: "Other places"?